tutorials/quickstarts/evals_quickstart.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
<p>I would probably just build a dataset in the notebook with say like 10 rows. It's more clear that way</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>